backend/Gemfile

```diff
@@ -39,6 +39,7 @@ gem "bootsnap", require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'rspec'
```

Comment:
We'll be using `rspec` but I would wait until we actually start writing tests to add the gem, there are also some other RSpec setup steps to do when you first introduce it.